lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -35,12 +39,24 @@ def to_native(self, course_overview):
             course_updates_url = None
             course_handouts_url = None
 
+        if course_overview.advertised_start is not None:
+            start_type = "string"
+            start_display = course_overview.advertised_start
+        elif course_overview.start != DEFAULT_START_DATE:
+            start_type = "timestamp"
+            start_display = defaultfilters.date(course_overview.start, "DATE_FORMAT")
```

Comment:
There exists a method `CourseOverview.start_datetime_text` that does start date formatting. If that suits this use case then you should use that for consistency; if not, then nevermind.